doc/src/thermo_modify.rst

```diff
@@ -75,6 +76,23 @@ are drifting out of the box through a fixed boundary condition (see
 the :doc:`boundary <boundary>` command).  In this case one atom may be
 deleted before the rest of the molecule is, on a later timestep.
 
+The *warn* keyword determines whether LAMMPS will print warning
+messages and how many of them.  Certain warning messages can be
+quite verbose and thus quickly blow up the size of the log file
+and screen output.  Thus a limit of 100 warnings messages is applied
```

Comment:
warnings -> warning.  Also explain whether N is cumulative over the lifetime of the input script or something else.